azure-pipelines.yml

```diff
@@ -14,139 +14,48 @@ pr:
     - '*'
 
 variables:
-  - name: is-runtime-release
-    value: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/Runtime-v')]
-  - name: is-sdk-release
-    value: $[startsWith(variables['Build.SourceBranch'], 'refs/tags/SDK-v')]
-  - name: Codeql.Enabled
-    value: true
+- name: Codeql.Enabled
+  value: true
+
+parameters:
+- name: pools
+  type: object
+  default:
+  - name: NetCore-Public
+    vmImage: ubuntu-latest # The public PR pool schema is different from the 1ES one. We use 'vmImage' which is a default azure pipeline schema.
+    os: linux
+  - name: NetCore-Public
+    vmImage: macOS-latest # 1ES uses 'image' but defining an image this way using our public pools does not work as they aren't configured this way.
+    os: macOS
+  - name: NetCore-Public
+    vmImage: windows-latest # We add the flag 'oes' or one engineering system: false or true in all tasks to determine if we use vmImage or a 1es pool object.
```

Comment:
Same here for the `NetCore-Public` images for Windows. I'd recommend: `windows.vs2022preview.amd64.open`